infinite-list/infinite-list.css

```diff
@@ -0,0 +1,39 @@
+body {
+  font-family: 'myriad pro', sans-serif;
+  font-size: 20px;
+  border: 0;
+  max-height: 100%;
+  overflow: hidden;
+  padding: 0;
+  margin: 0;
+}
+
+infinite-list{
+	display: block;
```

Comment:
Pls use 2 spaces instead of tab. There're some indentation errors in other files, too. Check it.